mgnify_pipelines_toolkit/analysis/assembly/gff_annotation_utils.py

```diff
@@ -23,13 +23,16 @@
     EVALUE_CUTOFF_IPS,
     EVALUE_CUTOFF_EGGNOG,
 )
+<<<<<<< HEAD
```

Comment:
These notations are conflicts that arise during the file modification. You have to solve it by selecting the correct code. Also, remove this git notation